.github/workflows/goreleaser-build-publish-develop.yml

```diff
@@ -4,9 +4,26 @@ on:
   push:
     branches:
       - develop
+  workflow_dispatch:
+    inputs:
+      git_ref:
+        description: "Git ref (commit SHA, branch name, tag name, etc.) to checkout"
+        required: true
+
+env:
+  GIT_REF: ${{ github.event.inputs.git_ref || github.ref }}
 
 jobs:
   push-chainlink-develop-goreleaser:
+    strategy:
+      matrix:
+        image:
+          - name: ""
```

Comment:
should this be named? like `core`?

Reply:
It's a good suggestion but we have this matrix elsewhere, so I think we should just keep it blank for now (as default).